GpuMiner/MinerEngine/CLMiner.cpp

```diff
@@ -443,11 +450,15 @@ bool CLMiner::Initialize()
         if(_globalWorkSize % _workgroupSize != 0)
         {
             _globalWorkSize = ((_globalWorkSize / _workgroupSize) + 1) * _workgroupSize;
-        }
-
-        //AddDefinition(_kernelCode, "PLATFORM", platformId);
-        AddDefinition(_kernelCode, "OUTPUT_SIZE", OUTPUT_SIZE);
-        AddDefinition(_kernelCode, "ITERATIONS_COUNT", KERNEL_ITERATIONS);
+        }
+        
+#if defined (__APPLE__)|| defined (__MACOS)
+        AddDefinition(_kernelCode, "__MACOS", 1);
```

Comment:
You have added new definition, but how do you use it?

Reply:
Good point! When I tested the code, i modified the kernel code. But now it useless. This definition should be removed.